generator/source/release/chime/notes.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: page
+hide: true
+title: Hyperledger Sawtooth 1.2 (Chime)
+permalink: /release/chime/
+release: 1.2
+release-name: chime
+# Copyright (c) 2019 Cargill Incorporated
+# Licensed under Creative Commons Attribution 4.0 International License
+# https://creativecommons.org/licenses/by/4.0/
+---
+
+<a href="top"></a>
+
+Hyperledger Sawtooth 1.2 (Chime) is now available.  See the [latest
+documentation](https://sawtooth.hyperledger.org/docs/core/releases/1.2.3/) to
```

Comment:
Hard coding release `1.2.3` to `latest` may be a problem when future point releases are made, such as `1.2.4`. Just something to think about.

Reply:
I have proposed internally that we create "release label" symlinks for bumper, chime, and so on.  We'll re-address these then.